README.md

```diff
@@ -86,11 +87,13 @@ environment variable
 
 ##### Steps for Windows platform
 
-* Go to directory with cloned repository
+* Go to directory with cloned repository and enter subfolder tools\ms
```

Comment:
@pvvasilev thanks for the corrections!
Fixed in [f1215b4](https://github.com/smartdevicelink/sdl_core/pull/791/commits/f1215b46dafe9cf25b752c45288ae111a6d1c156)